src/components/TopTabs.js

```diff
@@ -1,8 +1,11 @@
 import * as s from "./TopTabs.sc";
 import { TopTab } from "./TopTab";
+import CustomizeFeed from "../articles/CustomizeFeed";
 
 // Renders a title and the corresponding tabs links
-export default function TopTabs({ title, tabsAndLinks }) {
+export default function TopTabs({ title, tabsAndLinks, isSwipeView = false }) {
```

Comment:
The `SwipeView` flag propagating through multiple components is a sign that we're fighting the architecture. If you look at the description of the `TopTabs` component above this line, you will see that it is a generic component that renders titles and links. By making it aware of the `isSwipeView` we're making it "know too much". Let's define an `ArticleSwipePage` that defines it's own topTabs if they're different than the ones of the scrollable list by using this reusable component.